src/main/scala/com/tapad/docker/DockerComposeKeys.scala

```diff
@@ -24,6 +24,7 @@ trait DockerComposeKeysLocal {
   val testCasesPackageTask = taskKey[File]("The sbt TaskKey to package the test cases used when running 'dockerComposeTest'. This defaults to the 'packageBin' task in the 'Test' Scope.")
   val testDependenciesClasspath = taskKey[String]("The path to all managed and unmanaged Test and Compile dependencies. This path needs to include the ScalaTest Jar for the tests to execute. This defaults to all managedClasspath and unmanagedClasspath in the Test and fullClasspath in the Compile Scope.")
   val testPassUseSpecs2 = settingKey[Boolean]("True if Specs2 is to be used to execute the test pass. This defaults to False and ScalaTest is used.")
+  val testPassUseCucumber = settingKey[Boolean]("True if Cucumber is to be used to execute the test pass. This defaults to False and ScalaTest is used.")
```

Comment:
The Optional parameters section of the README should be updated to include this new setting as well:

```
Optional: There are a number of optional Keys that can be set as well if you want to override the default settings
...
```

Reply:
I've added an entry for the readme as well